### PR TITLE
Added requestId member on all CloudFrontEvent types

### PR DESCRIPTION
**Pull request by nhodin**
The requestId member is defined on all origin/viewer requests/responses, it is described here : https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-event-structure.html
There is one example for each request/response and they all have a cf.config.requestId.
Added new requests examples in tests in order to have all 4 types availabl

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-event-structure.html
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/aws-lambda/common/cloudfront.d.ts
Comment (simonbuchan) [inline types/aws-lambda/common/cloudfront.d.ts:63]: completely trivial nit: spacing around the middle `|`

**Review by simonbuchan — APPROVED** [review]
Looks good. I checked with the original PR that added them (#40355) to see the justification, and it specifically calls out the same documentation with "requestId is only available for viewer events", so it looks like either the docs were wrong or cloudfront has been updated since.

File: types/aws-lambda/common/cloudfront.d.ts
Comment (nhodin) [reply types/aws-lambda/common/cloudfront.d.ts:63]: Indeed, sorry... I am able to fix this before the merge ?

Comment (simonbuchan) [reply types/aws-lambda/common/cloudfront.d.ts:63]: It may require me to re-approve, but that's no problem.

Comment (nhodin) [reply types/aws-lambda/common/cloudfront.d.ts]: I made a commit for this : https://github.com/nhodin/DefinitelyTyped/commit/6564ad32b7a2e1a1eed06cb419b3cf26582bc970 . Should i make a new merge request ?

**Review by simonbuchan — APPROVED** [review]
Looks like the bot has changed a little, and you can merge it yourself now! It also looks like it's detected there's isn't another review necessary, but just in case.

**Comment by nhodin** [comment]
> Looks like the bot has changed a little, and you can merge it yourself now! It also looks like it's detected there's isn't another review necessary, but just in case.

Thanks ! But i don't find any Merge button in this page :( Is there another way to do it ?

**Comment by orta** [comment]
The bot is _very_ new - and we're working out the copy and issues ATM. Can try following the directions in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43696#issuecomment-610277519 please @nhodin

**Comment by nhodin** [comment]
Ready to merge

**Comment by nhodin** [comment]
@orta thanks for your help, sorry that I didn't see that comment earlier..

**Comment by orta** [comment]
You're welcome - there's a lot of noise, but I think this is the first from "0 -> Self Merge" we've seen.

**Comment by RyanCavanaugh** [comment]
🎉

**Pull request merged**
